File: benchmarks/sum-array/sum.js
const n = 10000000;
const arr = new Array(n);
for (let i = 0; i < n; i++) {
    arr[i] = i;
}

let sum = 0;
for (let i = 0; i < n; i++) {
    sum += arr[i];
}
console.log(sum);

File: benchmarks/sum-array/sum.go
package main

import "fmt"

func main() {
	n := 10000000
	arr := make([]int, n)
	for i := 0; i < n; i++ {
		arr[i] = i
	}

	sum := 0
	for i := 0; i < n; i++ {
		sum += arr[i]
	}
	fmt.Println(sum)
}
